src/components/HomePage/mainViewStyles.js

```diff
@@ -9,8 +9,12 @@ export default makeStyles((theme) => ({
     mainLogo: ({ isMobile }) => ({
         textAlign: "center",
         paddingBottom: "3em",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
         "& img": {
             width: isMobile ? "80%" : "50%",
+            maxHeight: isMobile ? "max(20vh, 6em)" : "unset",
```

Comment:
How did you arrive at these values ?

Reply:
20vh was the suggestion given in the linked issue. 6em is an arbitrary value I picked after experimenting a bit, to keep image from reducing its size too much in very small displays. I'll take suggestions.